default/scripting/specials/ASTEROID_COATING_OWNED.focs.txt

```diff
@@ -0,0 +1,30 @@
+Special
+    name = "ASTEROID_COATING_OWNED_SPECIAL"
+    description = "ASTEROID_COATING_OWNED_SPECIAL_DESC"
+    stealth = 5
+    spawnrate = 0.0
+    spawnlimit = 0
+    location = And [
+        Planet
+        Planet type = Asteroids
+        Not WithinStarlaneJumps jumps = 3 condition = And [
+            System
+            Contains And [
+                Planet
+                OwnedBy affiliation = AnyEmpire
+            ]
+        ]
+    ]
+    effectsgroups = [
+
+        // Gives a 10% structure bonus to all ships with asteroid hulls in the empire 
+        EffectsGroup
+            scope = And [
+                Ship
+                OwnedBy empire = Source.Owner
+                HasTag name = "ASTEROID_HULL"
+            ]
+            effects = SetMaxStructure value = Value + 0.1*abs(Value) accountinglabel = "ASTEROID_COATING_SPECIAL"
```

Comment:
Drop `abs`.  MaxStructure should be safely assumed as a positive value here (if not, other scripts may need revising).